lab7/tests/unit/test_line.py

```diff
@@ -1,5 +1,38 @@
-from lab7.line import Line
-from lab7.point import Point
+import math as m
+
+
+class Point:
+    def __init__(self, x: float, y: float):
+        self.x = x
+        self.y = y
+
+    def __str__(self):
+        return f'Point(x: {self.x}, y: {self.y})'
+
+
+class Line:
+    def __init__(self, point_a: Point, point_b: Point):
+        self.point_a = point_a
+        self.point_b = point_b
+
+    def __str__(self):
+        return f'Line: [{self.point_a}, {self.point_b}]'
+
+    def get_length(self):
+        return m.sqrt((self.point_b.x - self.point_a.x) ** 2 +
+                      (self.point_b.y - self.point_a.y) ** 2)
+
+    def get_middle_point(self):
+        x = (self.point_a.x + self.point_b.x) / 2
+        y = (self.point_a.y + self.point_b.y) / 2
+
+        return Point(x, y)
+
+    def get_length_projection_x(self):
+        return abs(self.point_b.x - self.point_a.x)
+
+    def get_length_projection_y(self):
+        return abs(self.point_b.y - self.point_a.y)
```

Comment:
How do I fix this??

Reply:
you don't need to redefine your class in a test file. Just import it

Reply:
It says "No module named 'line'" when i try to do it

Reply:
you should mention a whole path from the root folder

Reply:
just wait a bit, tomorrow we'll consider it

Reply:
ok